content/en/continuous_integration/setup_tests/ruby.md

```diff
@@ -12,29 +12,26 @@ further_reading:
 
 ## Compatibility
 
-Supported CI providers:
-* Appveyor
-* Azure Pipelines
-* BitBucket
-* BuildKite
-* CircleCI
-* GitHub Actions
-* GitLab
-* Jenkins
-* TravisCI
+Supported Ruby interpreters:
+* Ruby >= 2.1
+* JRuby >= 9.2
+
+Supported test frameworks:
+* Cucumber >= 3.0
+* RSpec >= 3.0.0
 
 ## Prerequisites
 
 [Install the Datadog Agent to collect tests data][1].
 
-## Installing tracing
+## Installing the Ruby tracer
 
 To install the Ruby tracer:
 
-1. Add the `ddtrace` gem (version 0.50.0 or above) to your `Gemfile`:
+1. Add the `ddtrace` gem to your `Gemfile`:
```

Comment:
Given the changes are available in the very latest version of `ddtrace`, wouldn't it be prudent to keep the minimum version here?

Unless we are having the pattern of not having the version number for any of our APM libraries, as long as the latest one supports the feature, then I think adding the minimum version is helpful.

Reply:
I've removed it from the step description but it's still on the `Gemfile` code below. I can readd it to the description though.